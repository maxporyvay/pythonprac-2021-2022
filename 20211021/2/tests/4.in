:maxFROMfour a b c d max(a,b,c,d)
maxFROMfour 2 0 9 -1
:uselessFun x y z s sin(x+y)/cos(z-s)
uselessFun pi/4 pi/4 pi/4 pi/4
quit "{} ... {}"