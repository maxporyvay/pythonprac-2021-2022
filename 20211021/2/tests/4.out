9
1.0
3 ... 5
